### PR TITLE
Fix Bug 1146474 - Update /en-US/firefox/sms/ With Legal Text & Link

### PR DESCRIPTION
**Pull request by kyoshino**
No l10n required. I would be nice to have it in today's push.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/android/sms-send.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h1 class="large">{{_('Get Firefox for Android')}}</h1>
       Send yourself a text message <br>to download it from Google Play.
     {% endtrans %}
     </h2>
-    <p class="note">{{ _('SMS service available in the U.S. only') }}</p>
+    <p class="note">{{ _('SMS service available in the U.S. only. SMS &amp; data rates may apply. The intended recipient of the SMS must have consented. <a href="%s">Learn more</a>')|format(url('privacy.notices.websites') + '#campaigns') }}</p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/android/sms-send.html]: Any reason to use `&amp;` here and not just `&`?

Comment (kyoshino) [inline bedrock/firefox/templates/firefox/android/sms-send.html]: Looks like other strings are using ` &amp;` and `&` is not escaped somehow.

Comment (alexgibson) [inline bedrock/firefox/templates/firefox/android/sms-send.html]: Hmm, seems to get escaped to me. But if other strings are using it then let's not hold up

Comment (kyoshino) [inline bedrock/firefox/templates/firefox/android/sms-send.html]: I guess the escape issue is site-wide. See https://www.mozilla.org/en-US/firefox/37.0/whatsnew/

**Pull request merged**